src/notes/notes.service.ts

```diff
@@ -117,27 +118,13 @@ export class NotesService {
    * @return {string} the content of the note
    */
   async getNoteContent(note: Note): Promise<string> {
-    return (await this.getLatestRevision(note)).content;
-  }
-
-  /**
-   * @async
-   * Get the first revision of the note.
-   * @param {Note} note - the note to use
-   * @return {Revision} the first revision of the note
-   */
-  async getLatestRevision(note: Note): Promise<Revision> {
```

Comment:
Why was this deleted?

Reply:
Because it was a unnecessary function that was only used in places that also import the revisions service. And also this method provoked a lot of cycling dependencies.